packages/image-block/src/ImageBlock.tsx

```diff
@@ -101,13 +101,14 @@ export const ImageBlock = ({ appBridge }: BlockProps) => {
         deleteAssetIdsFromKey(IMAGE_ID, [image?.id]);
     };
 
+    useEffect(() => {
+        blockSettings.name && setIsTitleUpdatable(false);
+    }, [blockSettings.name]);
+
     return (
         <div
             data-test-id="image-block"
-            className={joinClassNames([
-                'tw-flex tw-h-auto tw-gap-3',
-                mapCaptionPositionClasses[blockSettings.positioning],
-            ])}
+            className={`tw-flex tw-h-auto tw-gap-3 ${mapCaptionPositionClasses[blockSettings.positioning]}`}
```

Comment:
how come you removed joinClassnames?

Reply:
maybe personal preference but I do like the helper function because it is easily extendable with extra conditions/classnames if needed. Nonetheless it is a minor detail.